ticdc/ticdc-changefeed-config.md

```diff
@@ -111,16 +127,32 @@ dispatchers = [
 # When the downstream is Kafka, the protocol can only be canal-json or avro.
 # When the downstream is a storage service, the protocol can only be canal-json or csv.
+# NOTE: ignore this field when the downstream type is the DB.
 protocol = "canal-json"
 
 # The following three configuration items are only used when you replicate data to storage sinks and can be ignored when replicating data to MQ or MySQL sinks.
 # Row terminator, used for separating two data change events. The default value is an empty string, which means "\r\n" is used.
 terminator = ''
-# Date separator type used in the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. For more information, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records>.
+# Date separator type used in the file directory. Value options are `none`, `year`, `month`, and `day`. `none` is the default value and means that the date is not separated. For more information, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records>. NOTE: ignore this field when replicating data to the downstream other than Storage.
 date-separator = 'none'
-# Whether to use partitions as the separation string. The default value is true, which means that partitions in a table are stored in separate directories. It is recommended that you keep the value as `true` to avoid potential data loss in downstream partitioned tables <https://github.com/pingcap/tiflow/issues/8724>. For usage examples, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records)>.
+# Whether to use partitions as the separation string. The default value is true, which means that partitions in a table are stored in separate directories. It is recommended that you keep the value as `true` to avoid potential data loss in downstream partitioned tables <https://github.com/pingcap/tiflow/issues/8724>. For usage examples, see <https://docs.pingcap.com/tidb/dev/ticdc-sink-to-cloud-storage#data-change-records)>. NOTE: ignore this field when replicating data to the downstream with type other than Storage.
 enable-partition-separator = true
 
+# Schema registry url. NOTE: ignore this field if you replicate data to the downstream types other than MQ. 
+# schema-registry = "http://localhost:80801/subjects/{subject-name}/versions/{version-number}/schema"
+
+# Indicates number of encoder thread used when encoding data.
+# NOTE: ignore this field when the downstream type is not the MQ.
+# encoder-concurrency = 16
+
+# Indicates if enabling kafka-sink-v2 which uses kafka-go sink library.
+# NOTE: ignore this field when the downstream type is not the MQ.
+# enable-kafka-sink-v2 = false
+
+# If set only output the updated column.
+# NOTE: this field only apply to the MQ downstream using the open-protocol.
```

Comment:
This may not correct after this PR: https://github.com/pingcap/tiflow/commit/25b91204953bef5cb065d4421ba675bb1b9648c8

Reply:
Didn't notice we have this PR under going

Reply:
fixed